CONTRIBUTING.md

```diff
@@ -1,429 +1,255 @@
+##### Creating
 
-#### Special Commit Messages
+When creating a new module within this project, it is best to begin by identifying whether or not the new module will exist within one of the three types:
 
-These are commit messages that will have an impact on how the build pipeline behaves. They are not to be used without prior approval.
+* **Legacy Module**
+  * Module that will utilize the older tooling (such as [Ampersand](https://www.npmjs.com/package/ampersand) or [Babel](https://babeljs.io/))
+  * This is typically any module that will exist within `./packages/@webex`.
+  * [Legacy Module Example - Messaging Plugin](./packages/@webex/plugin-messages/)
+* **Modern Module**
+  * Module that will be built using newer technology.
+  * This is recommended for any module that will not need to access legacy dependencies (such as [Ampersand](https://www.npmjs.com/package/ampersand)).
+  * [Modern Module Example - Package Tools](./packages/tools/package/)
+* **Migration Module**
+  * Any module that will be migrated into this project from an external project repository.
+  * [Migration Module Example - Calling SDK](./packages/calling/)
 
-All of these commit messages should include an explanation for why you're using them. You'll need to commit with `-n` or `--no-verify` to bypass the commit message linter.
+Once a module type has been determined, follow the linked example modules for examples on creating the module.
 
-> For example
-> `git commit -m "docs(webex-core): [skip npm] - docs change" --no-verify`
+##### Building
 
-##### `[skip npm]`
+Building modules can be done by following common script semantics provided by [SCRIPTS.md](./SCRIPTS.md). The recommended tooling for each type of module will be listed below:
 
-This will run through all the GitHub Checks but will skip any version bumping, tagging, and subsequent publishing to npm after a pull request is merged.
+* **Legacy Module** - [Legacy Tools](./packages/legacy/tools/).
+  * All tooling is ran through the CLI via `webex-legacy-tools`.
+  * Use `webex-legacy-tools --help` to get started
+  * Use the [Legacy Module Example - Messaging Plugin](./packages/@webex/plugin-messages/) as a template.
+* **Modern Module** - Any modern tooling that has [localized configuration](./config/).
+  * Typically will use [TypeScript](https://www.typescriptlang.org/) with the modern [TypeScript configuration](./config/typescript/).
+  * Will extract documentation using [API Extractor](https://api-extractor.com/) with the modern [API Extractor Configuration](./config/api-extractor/).
+  * Will build documentation using [API Documenter](https://www.npmjs.com/package/@microsoft/api-documenter).
+* **Migration Module**
+  * Originally, the tooling that existed in the previous project will be consumed.
+  * Ultimately, should be migrated to use **Modern Module Tooling**.
 
-##### `[skip ci]`
+##### Testing
```

Comment:
@InteractiveTimmy  Would it make sense to add a note here about the browser-tests that to run all the browser tests we need to have both chrome and firefox installed.

Reply:
Yes, I will add a note about chrome and firefox install requirements

Reply:
I have added this to the requirements section.